tests/cluster/tests/29-slot-migration-replication.tcl

```diff
@@ -0,0 +1,351 @@
+#Tests that validate slot migration states are replicated propoerly
```

Comment:
Going to poke again to put stuff in the tests/unit/cluster so they run as part of the CI.